tests/integration-test/bpf/build_push_images.sh

```diff
@@ -33,13 +33,43 @@ docker build \
 docker push quay.io/bpfd-bytecode/tracepoint
 
 docker build \
```

Comment:
Do we have an issue for building and pushing these in actions?

Reply:
I thought we did, but I can't find it right now.  I'll do a little more searching and will add one if I can't find one.  However, we usually do this when a pr is merged, but, when you're adding new test cases, the images need to be available for the new test cases to pass.  So, I think you also need to be able to push them manually.